circles_8sym.py

```diff
@@ -1,80 +1,84 @@
 import random, sys 
-footer = f'</svg>'
-randColors = [1,  0]
-colours = {0:'black',
-           1:'white'}
+rand_colors = [1,  0]
+colors = {0: 'black',
+           1: 'white'}
 
-def drawCirc(x,y,width, height, color='black'):
-    r = height/2
-    cx = x+(r)
-    cy = y+(r)
+def draw_circle(x, y, width, height, color='black'):
+    r = height / 2
+    cx = x + r
+    cy = y + r
     return f'<circle cx="{cx}" cy="{cy}" r="{r}" fill="{color}" stroke-width="0"/>'
 
-def createTile(size):
-    invader = []
+def create_tile(size):
+    mosaic = []
     #only do the first n//2 rows (treating the first n//2 cols first)
-    for r in range(size//2+1):
-        tempRow = []
+    for r in range(size // 2 + 1):
+        temp_row = []
         #needed to flip black and white circles
-        wasBlack = 0
-        for c in range(size//2+1):
+        was_black = 0
+        for c in range(size // 2 + 1):
             #if on the horizontal or vertical centre or on the diagonal
             #try not to make it too heavy on the black
-            if wasBlack>2:
-                tempRow.append(1)
-                wasBlack = 0
-            else:
-                if r==c or c==size//2+1 or r==size//2+1 or r<c:
-                    col = random.choice(randColors)
-                    tempRow.append(col)
-                    if col ==0:
-                        wasBlack +=1
-                    else:
-                        wasBlack = 0
-                #bottom of diag
-                elif r>c:
-                    tempRow.append(invader[c][r])
-        for c in range(size//2,-1,-1):
-            tempRow.append(tempRow[c])
-        invader.append(tempRow)
-    for r in range(size//2,-1,-1):
-        invader.append(invader[r])
-    return invader
+            if was_black > 2:
+                temp_row.append(1)
+                was_black = 0
+                continue
+            #otherwise
+            if r == c or c == size // 2 + 1 or r == size // 2 + 1 or r < c:
+                col = random.choice(rand_colors)
+                temp_row.append(col)
+                if col == 0:
+                    was_black += 1
+                else:
+                    was_black = 0
+            #bottom of diag
```

Comment:
add `continue` here and then the next line can just be `if r > c:`, not `elif`. This makes every "branch" the same.